SourceX/DiabloUI/mainmenu.cpp

```diff
@@ -49,15 +49,14 @@ void mainmenu_Load(char *name, void(*fnSound)(char *file))
 
 void mainmenu_Free()
 {
-	mem_free_dbg(ArtBackground.data);
-	ArtBackground.data = NULL;
+	ArtBackground.Unload();
```

Comment:
P.s. Isn't it still best to free the struct also?

Reply:
The struct is on stack